src/main/java/Ava.java

```diff
@@ -0,0 +1,107 @@
+import java.util.Scanner;
```

Comment:
I like the good practice of not importing java.util.*, avoids potential namespace collisions!